docs/product-administration-and-configuration-guide/src/main/asciidoc/kie-server-accessing-runtime-data-proc.adoc

```diff
@@ -0,0 +1,15 @@
+[id='kie-server-accessing-runtime-data-proc']
+= Accessing Runtime Data from {CENTRAL}
+
+The `Process Instances`, `Tasks`, and `Process & Task Dashboard` pages of Business Central allow you to view the runtime data of the Intelligent Process Server. These runtime pages use the credentials of the currently logged in user to load data from the Intelligent Process Server. Therefore, to be able to view the runtime data on Business Central, ensure that the following conditions are met:
+
+* The user exists in the container running the Business Central application. This user must have `admin`, `analyst`, or `developer` roles assigned, in addition to the `kie-server` role, with full access to the runtime data.
+* The user exists in the container running the Intelligent Process Server and has `kie-server` role assigned.
+* The communication between Business Central and Intelligent Process Server is established. That is, the Intelligent Process Server is registered in the server controller, which is part of Business Central.
+* The `deployment.business-central.war` login-module is present in the `standalone.xml` configuration of the server running Business Central. For example:
+[source]
+----
+ <login-module code="org.kie.security.jaas.KieLoginModule" flag="optional" module="deployment.business-central.war"/>
```

Comment:
ditto

Reply:
leaving this since it's not in this story.